tenets/codelingo/go/writes-to-async-pointer/example.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"fmt"
+)
+
+type thing struct {
+	name string
+}
+
+func getNameFromPointer(t *thing) {
+	fmt.Println(t.name)
+}
+
+func getName(t thing) {
+	fmt.Println(t.name)
+}
+
+func setNameOnPointer(t *thing) {
+	t.name = "car"
+}
+
+func newThingPointer(name string) *thing {
```

Comment:
I don't think this has any explanatory power over using `&thing{name}` in the source directly